i18n/ru/common.csv

```diff
@@ -0,0 +1,11 @@
+empty, "А ну пшел отсюдова"
+otherError, "ОшЫбка в твоем ДНК"
+,
+ping, "понг"
+ping, "понг по русски"
+ping, "понг епта что я делаю"
+ping, "понг я пишу код в час ночи"
+,
+status, "ХАХААХХА\nУровень: ${lvl}\nОпыт: ${value}\nСледующий уровень: никогда"
```

Comment:
just to mention: `${key}`-syntax can be confusing, btw.
Because you can't do something like `${key.value}`, etc inside it, so you *must* know that it is only a string parsed in i18n and so on.

Reply:
need use something like https://github.com/WebReflection/backtick-template#es2015-backticks-for-es3-engines--